__tests__/utils.test.js

```diff
@@ -102,3 +115,29 @@ describe("formatComments", () => {
     expect(formattedComments[0].created_at).toEqual(new Date(timestamp));
   });
 });
+
+describe.only("GET Requests", () => {
+  describe("/api/topics", () => {
+    test("status 200: responds with an array of topic objects, each with 'slug' and 'description' properties", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toEqual([
```

Comment:
This works in this case, but later on you'll need to take a look at how to iterate through objects in your tests and make sure that each one matches a specific structure (e.g. each one has a description property which will be a string.)